doc/user/how-to/troubleshooting.md

```diff
@@ -128,6 +128,19 @@ For more context, see the [GitHub
 issue](https://github.com/nsidc/qgreenland/issues/515).
 
 
+### When I open QGreenland, I get the warning `This project file was created by a newer version of QGIS`
+
+For example:
+
+```
+This project file was created by a newer version of QGIS (3.28.8-Firenze) and could not
+be completely loaded.
+```
+
+We haven't observed any adverse effects from this warning. If you want to get rid of the
+message permanently, update to the latest release of a supported QGIS version.
```

Comment:
One problem here is that the flatpak distribution doesn't have the latest LTS.

I opened an issue: https://github.com/flathub/org.qgis.qgis/issues/329

It doesn't look _too_ hard to contribute this change, but I haven't had the time to think about it yet.